lib/SILOptimizer/Transforms/DeadObjectElimination.cpp

```diff
@@ -825,6 +836,40 @@ DeadObjectElimination::removeInstructions(ArrayRef<SILInstruction*> toRemove) {
   }
 }
 
+void DeadObjectElimination::salvageDebugInfo(SILInstruction *toBeRemoved) {
+  if (auto *SI = dyn_cast<StoreInst>(toBeRemoved)) {
```

Comment:
LLVM coding style would be

auto *SI = dyn_cast<StoreInst>(toBeRemoved);
if (!SI)
  return;
...

Reply:
We generally try to use early exists wherever possible.